### ISSUE TITLE
get rid of the local cloud cache folder (or move it to SD)

### ISSUE THREAD

**Issue by netphantm** [nextcloud/android#12569]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
I once found out that nextcloud uses a local cache folder for the cloud files.
I use nextcloud as a backup solution for the (usually large) media files on my phone, so to keep a local cache of the synched files is frustrating, cos it fills up my phone, doubling the space needed for the files in the cloud.

**Describe the solution you'd like**
A possibility to move this cache folder to the SD (at least) would be nice, better yet, just get rid of it. The files are already in two places, locally and the cloud, so why cache them again?

**Describe alternatives you've considered**
If there would be a way to use a deduplication FS on the phone, it wouldn't be such a big deal, but I don't suppose that is an option on android..

**Additional context**
See also my other feature request to archive files (delete them locally) after some specified time.
https://github.com/nextcloud/android/issues/12568

**Comment by dmiales** [comment]
@joshtrichards I completely agree.  The way the “on device” tab is implemented is terrible.  It should show not only downloaded files (which were not available locally), but also all local files that were uploaded to nextcloud.  And what is downloaded should not be sent to the cache folder, but to the real one, which the application can also track.  This is just a necessary feature

**Comment by krzkraw** [comment]
I use Poweramp and attached Nextcloud as storage. Nextcloud kept downloading and caching the files locally until it filled up my phone and rendered it useless! I could not even dial a number, imagine something like this happening during emergency situation...

It was not Poweramp's fault - files were downloaded by and to Nextcloud data folder. I was not syncing this folder, set it to be available locally etc. As Poweramp was scanning media files, Nextcloud was downloading them to my phone's storage

I use Nextcloud as cloud storage to not have to waste all of my local storage for media... Please implement cache size limit and/or cache disable switch. I self host and love Nextcloud but this is dealbreaking now :(

**Comment by SamuXzX** [comment]
I think I finally found the issue I'm having but I'm not sure. What I see is that Nextcloud keeps wasting a lot of space even if I remove the downloaded files by deleting them locally. I thought I could go delete the caches of the app, but I see that caches are actually pretty small and Nextcloud seems to keep files in a data folder inaccessible to me.
